omero/developers/python3-migration.rst

```diff
@@ -0,0 +1,108 @@
+Migration from OMERO 5.5 (Python 2) to OMERO 5.6 (Python 3)
+===========================================================
+
+This page serves as a collection of tips the OME team used to
+get through the migration to Python 3. This is not a complete
+guide but may serve as a useful starting point.
+
+For more information, please see a dedicated Python 3 page like
+http://python-future.org/.
+
+Futurize
+--------
+
+Installing `future` from Python 3 is now required for all OMERO
+Python components. This library comes with a tool, `futurize`
+which will perform many of the basic transformations needed to
+migrate Python 2 code to Python 3:
+
+::
+
+    futurize -0 your_file.py
+
+Add `-w` to update the file in place.
+
+`print()`
+^^^^^^^^^
+
+The most common transformation needed is adding parentheses around
+print statements since print is no longer a keyword.
+
+`dict.keys()`
+^^^^^^^^^^^^^
+
+The return value from the `keys()` method of dictionaries is of type
+`dict_keys` and no longer has methods like `sort()`. Wrap with a call
+to list if you need the previous behavior: `list(my_dict.keys())`.
```

Comment:
Similar for e.g. ```range(5)```.
Also sort(lambda) and list.map(lambda) have needed updating. Do we want to mention all these?

Reply:
Happy to have whatever issues added that were problematic for us which we can easily explain to others, but I assume it's more critical to get a version of this PR in early so we have the URLs to link to, then we can iterate on this list.